client/reader/stream/index.jsx

```diff
@@ -438,6 +503,22 @@ class ReaderStream extends React.Component {
 				/>;
 		}
 
+		if ( postKey.isCombination ) {
+			const handleClick = post => showSelectedPost( {
```

Comment:
I'm afraid they all have this issue then: https://github.com/Automattic/wp-calypso/pull/11875/files/757a735aed29c4813ea40a346dd5a0f95e29dc93#diff-52f7f0c436e060a64c7081104b31880aR523.

I'll fix both here